src/styles/utilities/typography.scss

```diff
@@ -1,24 +1,24 @@
 @import './fonts.scss';
 
 h2 {
-  font-family: $lora;
   font-size: 42px;
+  font-family: $lora;
   line-height: 48px;
 }
 
 h4 {
-  font-family: $poppins;
-  font-size: 21px;
   font-weight: 500;
-  letter-spacing: 0.02em;
+  font-size: 21px;
+  font-family: $poppins;
   line-height: 31px;
+  letter-spacing: 0.02em;
```

Comment:
this file i think should be double checked with the designs

Reply:
ie idk if design has letters spaced anywhere

Reply:
As of now the typography is a bit inconsistent and most of it is being overwritten. Might need to bring this up to designer. I can delete the letter-spacing for now.

Reply:
works for me